cdap-tms/src/main/java/co/cask/cdap/messaging/store/leveldb/LevelDBTableFactory.java

```diff
@@ -69,42 +81,76 @@ public LevelDBTableFactory(CConfiguration cConf) {
     executor.scheduleAtFixedRate(new DataCleanup(), 0L,
                                  Long.parseLong(cConf.get(Constants.MessagingSystem.LOCAL_DATA_CLEANUP_FREQUENCY)),
                                  TimeUnit.SECONDS);
+
+    // Currently we don't support customizable table name yet, hence always get it from cConf.
+    // Later on it can be done by topic properties, with impersonation setting as well.
```

Comment:
We won't have impersonation in LevelDB, right?

Reply:
right, moved it to the proper spot.